packages/thirdweb/src/exports/wallets.ts

```diff
@@ -98,9 +98,14 @@ export type {
 } from "../wallets/in-app/core/wallet/types.js";
 
 export {
+  preAuthenticate,
+  authenticate,
+  authenticateWithRedirect,
```

Comment:
this one does not exist in `native` but it will show up in types, would it make sense to have it in native and just do `authenticate` when called?

Reply:
i added it, but throwing instead.